### DISCUSSION TITLE
Run medusa backend using mysql

### DISCUSSION THREAD

**Discussion by zubairfayyaz** [medusajs/medusa#3731] [Q&A]
I not want to run medusa app in postgresql database. I know its default database for medusa js. But I want to run my medusa app using mysql. Any idea?

**Comment by olivermrbl** [comment]
We do not officially support Mysql as of now. It is on our radar to add compatibility with more databases, as our ORM allows for it, but it is not on our near-future roadmap - unfortunately.
